fmriprep/workflows/util.py

```diff
@@ -8,12 +8,59 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
 
 from nipype import logging
```

Comment:
Sorry, one last thing: Could you drop the `logging` import?

Since the workflow is built in docs, this can be a docs only commit, so we can fast track this.